server/services/educationAndWorkPlanService.test.ts

```diff
@@ -76,12 +78,27 @@ describe('educationAndWorkPlanService', () => {
       expect(actual).toEqual(expectedActionPlan)
     })
 
-    it('should not get Action Plan given educationAndWorkPlanClient returns an error', async () => {
+    it('should not get Action Plan given prisonService returns an error', async () => {
```

Comment:
I'm not sure by this scenario ... 🤔 

The service call is `getActionPlan`, and in the other test (which I guess you based this one on), if the `educationAndWorkPlanClient` returns an error then clearly we cant get the action plan, so returning an object with `problemRetrievingData` set make sense 👍 

But is the same true for if the `prisonService` returns an error? 🤔  The scenario would be that we have successfully retrieved the action plan (the core data of interest), but we cant augment/decorate it with the prison names. Is that an error (in the same way)? or do we just return the action plan with the prison IDs in it rather than the looked up names?

Based on what we've done elsewhere we would return the action plan with the prison IDs, rather than returning some error state.

What do you think?
(Oh, and FWIW I would not recommend asking the BAs, and definitely not the design team, about this! They will likely want to design some kind of "empty state" (I hate that term!) which will no doubt add delay and complexity. I think we should be able to make a decision between us about what to do in this circumstance 👍 )